app/views/users/new.html.erb

```diff
@@ -1,24 +1,47 @@
 <h1>Registration</h1>
-
-    <%= form_for(@user) do |f| %>
-
-      <%= f.label :name %><br>
-      <%= f.text_field :name %><br>
-
-      <%= f.label :phase %><br>
-      <%= f.select :phase, options_for_select([0, 1, 2, 3, 4]) %><br>
-
-      <%= f.label :is_mentor, "Roll" %><br>
-      <%= f.radio_button :is_mentor, true %> Mentor <br>
-      <%= f.radio_button :is_mentor, false %> Student <br>
-
-      <%= f.label :email %><br>
-      <%= f.email_field :email %><br>
-
-      <%= f.label :password %><br>
-      <%= f.password_field :password %><br>
-      <%= f.password_field :password_confirmation %><br>
-
-      <%= f.submit "Sign Up" %>
-    <% end %>
-
+	<div class="form-horizontal">
+	 <fieldset>
+
+		<%= form_for(@user) do |f| %>
+		<div class="form-group">
+			<%= f.label :name, class:"col-lg-2 control-label" %><br>
+			<div class="col-lg-10">
+			<%= f.text_field :name, class: "col-lg-2 control-label" %><br>
+			</div>
+		</div>
+ 		<div class="form-group">
+			<%= f.label :phase, class:"col-lg-2 control-label" %><br>
+			<div class="col-lg-10">
+			<%= f.select :phase,options_for_select([0, 1, 2, 3, 4]),class:"form-control" %><br>
+			</div>
+		</div>
+ 		<div class="form-group">
+			<%= f.label :is_mentor, "Roll", class:"col-lg-2 control-label" %><br>
+			<div class="col-lg-10">
+			<%= f.radio_button :is_mentor, true %> Mentor <br>
+			<%= f.radio_button :is_mentor, false %> Student <br>
+			</div>
+		</div>
+		<div class="form-group">
+			<%= f.label :email, class:"col-lg-2 control-label" %><br>
+			<div class="col-lg-10">
+			<%= f.email_field :email, class: "col-lg-2 control-label" %><br>
+			</div>
+		</div>
+ 		<div class="form-group">
+ 			<div class="col-lg-10">
+ 			<%= f.label :password, class:"col-lg-2 control-label" %><br>
+			<%= f.password_field :password, class: "col-lg-2 control-label" %><br>
+			</div>
+			<div class="form-group">
+				<div class="col-lg-10">
+				<%= f.label :password_confirmation, class:"col-lg-2 control-label" %><br>
+				<%= f.password_field :password_confirmation, class: "col-lg-2 control-label" %><br>
+				</div>
+			</div>
+		</div>
```

Comment:
I'm not sure if @jdagley77 was working on this or @yaritzape9 but The password_confirmation field is out of alignment with the rest. See if you can get that fixed and push the branch up again. I'll leave this PR open and then merge it when I see it's been added.